creusot/src/translation/function.rs

```diff
@@ -377,31 +621,141 @@ impl<'body, 'tcx> BodyTranslator<'body, 'tcx> {
         id
     }
 
-    fn resolve_locals(&mut self, mut locals: BitSet<Local>) {
-        locals.subtract(&self.erased_locals);
+    fn resolve_places(
+        &mut self,
+        to_resolve: BitSet<MovePathIndex>,
+        resolved: &BitSet<MovePathIndex>,
+        assume_downcasts: bool,
+    ) {
```

Comment:
Just for future sanity, maybe add that this coalesces resolutions for places as a doc comment.